heat/core/statistics.py

```diff
@@ -1068,6 +1083,73 @@ def minimum(x1, x2, out=None):
     return lresult
 
 
+def __moment_w_axis(function, x, axis, elementwise_function, unbiased=None, Fischer=None):
```

Comment:
Can we have parameter and return type annotations here (except DNDarray due to aliasing)